app/src/main/java/com/door43/translationstudio/core/TargetTranslation.java

```diff
@@ -969,8 +969,9 @@ public void setPublished(final OnPublishedListener listener)  {
         try {
             Git git = getRepo().getGit();
             final TagCommand tag = git.tag();
-            SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd_HH.mm.ss", Locale.US);
```

Comment:
we want to keep the us locale because the output of this is machine readable. So we don't want to get any crazy symbols in the date.